meta-mel/conf/distro/include/mel.conf

```diff
@@ -1,8 +1,14 @@
 ## MEL Base Configuration {{{1
 DISTRO = "mel"
 ADE_DISTRO = "mel-flex-os"
-DISTRO_NAME = "Mentor Embedded Linux"
+OS_NAME = "Flex OS"
+GENERIC_NAME = "Mentor Embedded Linux"
+DISTRO_NAME = "${GENERIC_NAME} ${OS_NAME}"
+DISTRO_CODENAME = "dunfell"
```

Comment:
We may want to create an issue to create a bbclass which adds a sanity check -- make sure DISTRO_CODENAME is listed in LAYERSERIES_CORENAMES. This will ensure that we don't forget to update the DISTRO_CODENAME variable when updating to newer releases.